ims-examples/ob-connect_v1p0_draft-local.html

```diff
@@ -0,0 +1,992 @@
+<!DOCTYPE html lang="en">
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <script src='https://purl.imsglobal.org/spec/respec-ims-default.js' async class='remove'></script>
```

Comment:
Should this be a relatively sourced file? If not, perhaps the purl location needs to be updated (if it hasn't already)?